docs/usage.md

```diff
@@ -94,29 +96,39 @@ Bristlemouth can be integrated into other build systems by...
 <!--- Show how other build systems can utilize bm--->
 
 ## Running The Bristlemouth Stack
-Once Bristlemouth has been added to the project,
-it can be integrated within the project by configuring the following components:
-
-<!--- TODO: Refine this--->
-- Network Interface
-- Bristlemouth
 
-These items are configured by calling their initialization functions.
-An example of this and how the Bristlemouth stack is to be serviced is shown below:
+Once Bristlemouth has been added to the project build,
+the code integration is very simple: a single call to `bristlemouth_init`.
 
 ```C
-Adin2111 adin;
-Bristlemouth bm;
-
-int main() {
-    // other sensor setup
-    adin->init(...pins...spi peripheral...);
-    bm->init(adin, config);
-    for(;;) {
-        // other sensor code
-        bm->update();
+#include "bristlemouth.h"
+
+void network_device_power_callback(bool on) {
+    // Provide or cut off power to the network chip.
+    // On the mote, we set pin PH1 high or low.
+}
+
+int main(void) {
+    // ... your other setup code...
+
+    BmErr err = bristlemouth_init(network_device_power_callback);
+    if (err != BmOK) {
```

Comment:
Updated usage docs here, and made it a reality in https://github.com/bristlemouth/bm_protocol/pull/209!